bin/syclcc-clang

```diff
@@ -528,7 +549,8 @@ class pure_cpu_compiler:
     hipsycl_library_path = os.path.join(config.hipsycl_installation_path,"lib/")
 
     self._linker_args = []
-    self._compiler_args = ["-fopenmp", "-std=c++14"]
+    self._compiler_args = ["-fopenmp"]
+    self._common_compiler_args = config.common_compiler_args
```

Comment:
You're not actually using `self._common_compiler_args` for the CPU compiler, right? So, you would need to either also append it to `command` in `run()` or (I think preferably) just do `self._compiler_args += config.common_compiler_args` here.

Reply:
Yes, you're right.